File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program92.cpp
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept character from user. 
//		If character is small display its corresponding capital character, and if it small then display its corresponding capital. 
//		In other cases display as it is.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

class Solution
{
	private:
		char ch;
		
	public:
		Solution(char ch='\0')
		{
			this->ch = ch;
		}

		void Display()
		{
			if(ch>='a' && ch<='z')
			{
				cout<<char(ch-32)<<"\n";
			}
			else if(ch>='A' && ch<='Z')
			{
				cout<<char(ch+32)<<"\n";
			}
			else
			{
				cout<<ch<<"\n";
			}

		}


};

int main()
{
	char ch='\0';

	cout<<"Enter the character: ";
	cin>>ch;

	Solution *dObj = new Solution(ch);
	dObj->Display();
	
	delete dObj;
	return 0;
}

/*
OUTPUT:
Enter the character: Q
q

Enter the character: a
A

Enter the character: %
%

*/

File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program92.java
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept character from user. 
//		If character is small display its corresponding capital character, and if it small then display its corresponding capital. 
//		In other cases display as it is.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{

	public char ch;

	public Solution(char ch)
	{
		this.ch = ch;
	}

	public void Display()
	{	
		
		if(ch>='A' && ch<='Z')
		{
			System.out.println((char)(ch+32));
		}

		else if(ch>='a'&& ch<='z')
		{
			System.out.println((char)(ch-32));
		}
		else
		{
			System.out.println(ch);
		}
	}


}

class Program
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);

		System.out.println("Enter character:");
		char ch = sObj.next().charAt(0);

		Solution obj = new Solution(ch);
		obj.Display();
	}
}

/*
OUTPUT:
Enter character:
Q
q

*/
